picasso/src/main/java/com/squareup/picasso/Dispatcher.java

```diff
@@ -317,7 +321,7 @@ void performResumeTag(Object tag) {
     }
   }
 
-  void performRetry(BitmapHunter hunter) {
+  void performRetry(final BitmapHunter hunter) {
```

Comment:
I was using a visitor in this method at one point, this just didn't get cleaned up when I removed it.